sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/OrcReadBenchmark.scala

```diff
@@ -294,6 +294,38 @@ object OrcReadBenchmark extends SqlBasedBenchmark {
     }
   }
 
+  def structBenchmark(values: Int, width: Int): Unit = {
```

Comment:
Thank you. Could you spin off this benchmark addition into a separate PR, @bersprockets ?
We can merge it first.

Reply:
Will do.

Reply:
@dongjoon-hyun 
Created pull request for new benchmarks: #35100. I also added a benchmark for structs contained by arrays (to test whether copying rows is any faster or slower than creating new rows), as requested by @HyukjinKwon